hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationThread.java

```diff
@@ -35,7 +37,9 @@
     private final MigrationManager migrationManager;
     private final MigrationQueue queue;
     private final ILogger logger;
+    /** Time to sleep after {@link MigrateTask} */
```

Comment:
I updated the javadoc. I'm would rather not update any code in this PR, just add javadoc